src/components/table/draggable-table-cell/draggable-table-cell.js

```diff
@@ -19,12 +19,15 @@ const iconHTML = (
  * @constructor
  */
 const DraggableTableCell = (props) => {
+  const canDrag = props.canDrag !== false;
+
   const icon = (
     <WithDrag
       identifier={ props.identifier }
       draggableNode={ props.draggableNode }
+      canDrag={ canDrag }
     >
-      { iconHTML }
+      { canDrag ? iconHTML : <span /> }
```

Comment:
What's the purpose of this `span`?

Reply:
To make sure the row isn't being rendered with an 'icon', but instead is just empty